EMIP/1-1000/EMIP00124.xml

```diff
@@ -151,6 +152,63 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                         </seg>
+                    </handNote>
+                  </handDesc>
+
+                  <decoDesc>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#iv(erso)"/><desc><ref type="authFile" corresp="AT1031VirginAndChild"></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d02"><locus target="#iir(ecto)"/><desc><ref type="authFile" corresp="AT1082Angel">, with <term key="sword"/> <term key="scabbard"/> </ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#iiv(erso)"/><desc><ref type="authFile" corresp="AT1066DavidHarp"></ref> with caption “David, King and Prophet.”</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d04"><locus target="#21r"/><desc><ref type="authFile" corresp ="AT1086StrikingHead"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d05"><locus target="#50v"/><desc><ref type="authFile" corresp="AT1035HolyMan"> <persName ref="PRS4585Georgeo"/> and the <term key="dragon"/> </ref> with caption “Saint George.”</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d06"><locus target="#51r"/><desc><ref type="authFile" corresp="AT1082Angel"> with <term key="sword"/> <term key="scabbard"/></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d07"><locus target="#91r"/><desc><ref type="authFile" corresp ="AT1049Resurrection"/> with caption (l. to r.): “Eve,” and “How our Lord Jesus Christ rose from the dead on the third day.”</desc></decoNote>
+                    <decoNote type="miniature" xml:id="d08"><locus target="#117r"/><desc><ref type="authFile" corresp="AT1024MosesLaw"></ref> with caption “Prophet Moses with his tablet</desc></decoNote>
+                  </decoDesc>
+                  <additions>
+                  <list>
+                    <item xml:id="a01">
+                    <desc type="ScribalNoteCommissioning">
+                      ‎<locus target="#116v"/>
+                       Commissioned by <foreign xml:lang="gez">ክንፈ፡ ሚካኤል፡</foreign>
+                    </desc>
+                    </item>
+                    <item xml:id="a02">
+                    <desc type="ScribalSignature">
+                      ‎‎<locus target="#116v"/>
+                      Scribe’s name: <foreign xml:lang="gez">ተክለ፡ ማርያም፡</foreign>
+                    </desc>
+                    </item>
+                    <item xml:id="a03">
+                    <desc type="ScribalNoteAssigningLand">
+                      ‎‎‎<locus target="#117v"/> Note of land inheritance, giving boundaries of the ‎property, in Amharic
+                    </desc>
+                    </item>
+                    <item xml:id="a04">
+                    <desc>
+                      ‎<locus target="#ir"/> pen trials‎
+                    </desc>
+                    </item>
+                    <item xml:id="e01">
+                    <desc>
```

Comment:
My feeling is that this information belongs in //foliation or in //collation/signatures. @thea-m what do you think? https://betamasaheft.eu/Guidelines/?id=collationDesc

Reply:
Yes, //collation/signatures is the right place for information on quire marks.